packages/iota/test/validation/features.spec.ts

```diff
@@ -8,9 +8,11 @@ import { SENDER_FEATURE_TYPE } from "../../src/models/features/ISenderFeature";
 import { TAG_FEATURE_TYPE } from "../../src/models/features/ITagFeature";
 import { validateFeatures } from "../../src/validation/features/features";
 
+const MAX_FEATURES_COUNT = 4;
```

Comment:
Why we redefine this const here ?

Reply:
This is not redefined. It is only defined here to pass in `validateFeatures` function.